client/src/components/pages/Signup/index.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import './index.css';
+import UnderConstruction from '../UnderConstruction';
+
+const Signup = () => <UnderConstruction />;
```

Comment:
it feels the UnderConstruction component belongs more into the `components/common` directory than `components\pages` (doesn't have any state)